### PR TITLE
Add code to show current system user name and ID

### PR DESCRIPTION
**Pull request by sterndata**

### CODE REVIEW

**Comment by Clorith** [comment]
I think we should declare this a private field, as many will see the disclosure of system users and groups as a liability, and marking it as private means it then won't included in the automated copy-paste field.

**Comment by sterndata** [comment]
No objection to that. Do you want to edit on your end, or do you want me to fix and re-do a PR?

**Comment by Clorith** [comment]
You can just update your fork and the PR will self-update when you commit it 👍

File: src/includes/class-health-check-debug-data.php [hunk 1]
```diff
@@ -392,6 +397,11 @@ static function debug_data( $locale = null ) {
 			'label' => __( 'PHP SAPI', 'health-check' ),
 			'value' => ( ! function_exists( 'php_sapi_name' ) ? __( 'Unable to determine PHP SAPI', 'health-check' ) : php_sapi_name() ),
 		);
+		$info['wp-server']['fields'][] = array(
+			'label'   => __( 'PHP User', 'health-check' ),
+			'value'   => get_current_user() . ' (' . getmyuid() . ')',
```
Comment (Clorith) [inline src/includes/class-health-check-debug-data.php]: I had a thought here, it's not unlikely that these kind of functions might be disabled on a host level in various scenarios, we should add a `function_exists` wrapper to this check to ensure we don't call a fatal error in such cases.

Comment (sterndata) [reply src/includes/class-health-check-debug-data.php]: I wrapped it in a test for the two functions involved.

**Comment by Clorith** [comment]
Would you please change the branch to `develop` before this can be implemented, to suit the plugins new development flow.

**Comment by sterndata** [comment]
It seems o have been fixed already in the develop branch. A search does not find "installer".

**Comment by Clorith** [comment]
I'm not sure what your last comment here references, do you have this PR confused with another perhaps?

**Comment by Clorith** [comment]
Added to `develop` in 14c1958c5f3a02b3c23c7f4341f464098c5a58b8

**Pull request closed without merging**